### DISCUSSION TITLE
Alguém conhece a FIAP? É realmente diferente ou mais do mesmo com um bom marketing?

### DISCUSSION THREAD

**Discussion by oMatheus16** [frontendbr/forum#2065] [Perguntas e Respostas]
Eu estou pensando se vou ou não fazer faculdade e, se sim, em qual eu vou estudar. Há alguns dias me deparei com a FIAP e achei incrível, vários dias desde então pensando em estudar lá, querendo saber mais e aí, hoje, eu fui olhar as avaliações do Google e tive uma surpresa. Quando olhei o Instagram piorou kkkkk As avaliações são péssimas, principalmente quando se diz respeito ao atendimento. Outra coisa é que eu vi alguns comentários falando que a didática é a mesma coisa monótona das faculdades tradicionais, um PPT e o professor falando (E isso é tudo que eu menos quero), isso vai totalmente ao contrário daquilo que eles apresentam. Os únicos comentários realmente positivos são de 4/5 anos atrás, tem alguém estudando lá agora? Como foi a experiência de vcs quanto a isso?

**Comment by daianefernandes** [comment]
Olá Matheus, tudo bem?
Bom, vou dar minha opinião sincera sobre a FIAP. 

Já fiz um MBA lá e atualmente estou cursando uma graduação online. As duas foram completamente diferentes em experiência.
O diferencial da FIAP é sem dúvida com relação a infraestrutura e a qualidade de professores e coordenadores.

Não sei se você está buscando algo presencial ou online, mas se você não quer o padrão de mercado, sugiro o online onde você fica mais 'livre', com conteúdos em vídeo, podcast e pdfs.
Na FIAP eles não aplicam provas, as notas são baseadas em trabalhos que você tem que entregar praticamente toda semana e o detalhe de tudo é o TCC, que não existe! No final do curso você vai ter criado uma startup e se for ganhador o prêmio é uma viagem para Babson por conta deles.
Conheço algumas pessoas que ganharam o Startup One (como eles chamam) e no final da faculdade ficaram um período trabalhando só focado neste projeto que virou trabalho.
Para a área de tecnologia, acredito que seja uma das principais instituições, porque eles prezam muito por entregáveis e isso é bem incomum.

Não sei se consegui te ajudar, mas essa é a minha visão.

Boa sorte aí =)

**Comment by carolinagomss** [comment]
Se apresentam como uma instituição disruptiva (termo usado incansavelmente por eles) entretanto o marketing é mais forte e o ensino é padronizado. Estudo lá atualmente, tive conhecidos que fizeram faculdade/MBA e há anos continuam com conteúdos padronizados sobre coisas que não são utilizadas no mercado. Em contra partida, existem mesmo esse posicionamento por entregáveis, com casos que são trazidos por empresas de fora, mas tive o desprazer de vivenciar isso e no final só levarem em consideração a empresa e não os alunos que estudavam lá, inclusive meu grupo não teve retorno algum da empresa externa, somente dos professores e nos sentimos completamente desvalorizados e com tempo desperdiçado. Agora, se você necessita de um ensino presencial e depende de ajuda com recursos, eles conseguem ter uma ótima cobertura e disponibilizam computador para cada aluno, sem necessidade de dividir ou ter o seu próprio. Sobre as aulas, alguns conteúdos são com PPT, geralmente voltados para parte conceitual, outros professores colocam a mão na massa e te incentivam a fazer o mesmo, dão total disponibilidade para ajudar e recomendações sobre conteúdos extras. No momento as provas acontecem por entregas de trabalhos, que englobam estudo de caso em cima de uma situação real, com critérios de aceite para cada disciplina, anteriormente a pandemia, era aplicado prova de cada disciplina individual, fora trabalhos e um trabalho com peso maior por semestre. Outro ponto a ser considerado, as disciplinas são anuais e não semestrais como a maioria das faculdades, o que dificulta entrada no meio do curso (o que ocorreu na pandemia, já que havia muito aluno procurando a FIAP para migrar pois outras instituições não estavam preparadas para ensino a distância).

**Comment by thiagodsantos** [comment]
Não se engane por ser um site/plataforma com visual atrativo e se denominando a "Melhor faculdade de tecnologia do Brasil", algo que é totalmente errado, pois existem muito mais insitiuições melhores de tecnologia no país. E digo com propriedade pois atuo com desenvolvimento de sofware desde 2012.

Após 2 anos pagando regularmente a faculdade, e não querendo mais avançar no curso devido a baixa qualidade das lives e pouco conteúdo no portal para estudo, a faculdade irá te cobrar o curso todo, mesmo estando com a matrícula cancelada. Veja no Jusbrasil a quantidade de processos que a faculdade faz com seus alunos e ex-alunos, é só procurar por VSTP Educação no Jusbrasil e você encontrará mais de 5 mil processos.

Depois de pagar mais de 16 mil reais, ainda estão me cobrando mais de 22 mil reais, e alegando várias vezes, via email e WhatsApp que posso ter a conta bancária e bens bloqueados. E quando há momento que você não consegue contato direito com a instituição.

Mesmo que você tente fazer um acordo, eles rejeitam todas as propostas cabíveis de pagamento, pois querem retirar o máximo de dinheiro possível de você.

Recomendo que você procure outras instituições, seja EAD ou presencial que não seja da FIAP. A instituição só possui "nome" e "status" e não preza por ensinar o aluno, diferentemente de outras instituições do país, que prezam pelo aluno e que querem ver o progresso de seus alunos.

Corra dessa instituição, procure outras para não ter o que eu e outros milhares de alunos passam/passaram.

**Reply by alanasantosn9-hash** [reply]
olá, estou querendo ingressar na área de tecnologia (sistemas de informação), pretendo fazer um curso ead, e estava pensando na FIAP, depois de alguns comentários mudei minha opnião sobre ela, qual/quais faculdades vc/vcs me indicariam?

**Comment by EduardoGianisela** [comment]
Para quem está pesquisando faculdades e pensando em pagar a mensalidade altíssima da FIAP achando que vai sair de lá um cientista da computação de elite, aqui vai um choque de realidade.

O curso sofre de uma superficialidade extrema. Se você quer construir uma fundação sólida de baixo nível, fuja. Os problemas mais graves:

Cronograma Diluído: Chegamos ao 9º mês e a ementa de 'Computer Science' ainda está presa no básico de portas lógicas. Eles diluem o que bacharelados mais sérios liquidam em três semanas ao longo de quase um ano letivo.

Professores com base fraca: Na disciplina de 'Computer Science', presenciei o professor afirmando categoricamente que a regra distributiva não vale para o operador OU: A ∨ (B ∧ C) ↔ (A ∨ B) ∧ (A ∨ C). É um erro crasso, dado que, segundo Andrew S. Tanenbaum, a Lei Distributiva vale tanto para E quanto para OU.

Descompasso e Inadequação: Em 'Prompt And Artificial Intelligence', a desorganização é total. Nos primeiros seis meses, um dos professores entregou material de modelagem de dados (fora da ementa). Outro professor da mesma matéria teve a postura um tanto questionável de se gabar em sala que abandonou a faculdade anterior (Mackenzie) porque 'lá só tirava zero'.

Terceirização Cognitiva: O rigor acadêmico é zero. Todas as atividades permitem consulta e uso liberado de ChatGPT ou Gemini. Você passa sem precisar pensar.

A faculdade mascara o desnível da turma deixando a máquina fazer a prova por eles. O ambiente protege a mediocridade em vez de forçar o aluno a estudar.

**Comment by Granda-prog-create** [comment]
Vou falar por experiência própria, pois fiz ADS presencial e Engenharia de Sotware EAD.

A DIFERENÇA É BRUTAL!
No presencial a FIAP é, sem dúvida, a melhor faculdade de tecnologia do país. Já no EAD, é muito semelhante a um curso de Uniesquina. Lives semanais morosas e professor lendo slides, desafios que o ChatGPT faz sem nenhuma dificuldade e, por não ter provas, os trabalhos são bem fáceis, sempre focados em fazer um diagrama de como seria sua startup. 
Se for fazer presencial, recomendo, pois o contato com os professores e alunos é diário, fora o networking. No presencial, todas as atividades têm supervisão do professor, o que já ajuda bem mais. 
Já no EAD, a comunicação fica bem solta, pois são milhares de mensagens no chat do Teams e, na maioria das vezes, os professores nem conseguem te responder. 
Se você é um cara  pleno pra cima, não recomendo FIAP EAD. Você vai, literalmente, estar na turma de uma galera tão iniciante fazendo perguntas do tipo: "O que é uma classe? Por que usa if e else? Fazendo faculdade consigo estágio na Google?".
